docs/guides/server/db.adoc

```diff
@@ -128,20 +128,27 @@ Then continue configuring the database as described in the next section.
 
 For each supported database, the server provides some opinionated defaults to simplify database configuration. You complete the configuration by providing some key settings such as the database host and credentials.
 
-. Start the server and set the basic options to configure a database
+The configuration can be set during a *build command* OR a *start command*:
+
+. Using a *build command* followed by an optimzed start command (recommended)
++
+----
+bin/kc.[sh|bat] build --db postgres --db-url-host mypostgres --db-username myuser --db-password change_me
+bin/kc.[sh|bat] start --optimized
+----
++
+
+These commands include the minimum settings needed to connect to the database.
+
+. Using *only a start command* (without --optimized)
 +
 <@kc.start parameters="--db postgres --db-url-host mypostgres --db-username myuser --db-password change_me"/>
 +
 
 This command includes the minimum settings needed to connect to the database.
 
-The default schema is `keycloak`, but you can change it by using the `db-schema` configuration option.
+The default schema is `keycloak`, but you can change it by using the `--db-schema` configuration option.
```

Comment:
```suggestion
The default schema is `keycloak`, but you can change it by using the `db-schema` configuration option.
```

Since this can be set in other ways, so it doesn't need to be specific to the cli option.

Reply:
OK